### DISCUSSION TITLE
有经验的能说说：CF自选IP 用IPV6会更快吗？

### DISCUSSION THREAD

**Discussion by idragonet** [XIU2/CloudflareSpeedTest#347] [讨论]
会更快的话，搞定之家广东联通宽带的IPV6.

**Comment by XIU2** [comment]
这个我也不是很清楚。
因为我申请**公网 IPv4** 后，所以 **IPv6 就没了**（二选一）。
而我开发 IPv6 测速功能时，也是在我的服务器上测试的。

因此我平时都是在用 IPv4，自选 IPv6 是否能比默认解析 IPv6 更快，这个我没测试过。

**Comment by immity** [comment]
不会，除非是教育网

**Comment by CrazyBoyFeng** [comment]
以前用过南方移动家宽，它的 IPv6 出国比 IPv4 快。  
具体自己测一测就知道了。

不知道有没有人有“同时测 IPv4 和 IPv6”的需求？

**Comment by christarcher** [comment]
就IPv6目前的总出口带宽来看 比IPv4小得多

**Comment by CrazyBoyFeng** [comment]
> 就IPv6目前的总出口带宽来看 比IPv4小得多

出口小，但是用户也少，平均分到的带宽就不少了。华中电信家宽的测速 IPv4 和 IPv6 对比：  
|IP 地址|已发送|已接收|丢包率|平均延迟|下载速度 (MB/s)|
|----|----|----|----|----|----|
|2606:4700:3002:bd66:4ded:1c67:3308:2ceb|4|4|0|163.5|0.47|
……
|2606:4700:3005:6c3c:d16a:4c89:c9ee:bf39|4|4|0|163.25|0.37|
|104.17.14.120|4|4|0|170.83|0.31|
……
|104.17.87.168|4|4|0|177.55|0.09|

可以看到，在我这里，最慢的 IPv6 也比最快的 IPv4 快。

**Comment by christarcher** [comment]
> > 就IPv6目前的总出口带宽来看 比IPv4小得多
> 
> 出口小，但是用户也少，平均分到的带宽就不少了。华中电信家宽的测速 IPv4 和 IPv6 对比：
> 
> IP 地址	已发送	已接收	丢包率	平均延迟	下载速度 (MB/s)
> 2606:4700:3002:bd66:4ded:1c67:3308:2ceb	4	4	0	163.5	0.47
> ……					
> 2606:4700:3005:6c3c:d16a:4c89:c9ee:bf39	4	4	0	163.25	0.37
> 104.17.14.120	4	4	0	170.83	0.31
> ……					
> 104.17.87.168	4	4	0	177.55	0.09
> 可以看到，在我这里，最慢的 IPv6 也比最快的 IPv4 快。

ipv4怎么会这么慢？我三网跑下来都是ipv4快的可以到20多mb ipv6 1-2mb的样子

**Comment by CrazyBoyFeng** [comment]
跟所在地有关。全国并不是共用一个出口，即使共用一个出口的地区，也不一定是均分的。

**Comment by Anshi84** [comment]
北方移动，ipv4测速能跑满百兆带宽，ipv6只有不到2MB/s。
